index.js

```diff
@@ -106,6 +106,11 @@ const useYarn = () => fs.existsSync(yarnFilename)
 const usePnpm = () => fs.existsSync(pnpmLockFilename)
 const useNpm = () => fs.existsSync(packageLockFilename)
 
+let runPrefix = 'npx'
+if (useYarn()) {
+  runPrefix = 'yarn'
+}
+
```

Comment:
```suggestion
const runPrefix = useYarn() ? 'yarn' : 'npx'
```

Same suggestion here.

Reply:
@warrensplayer 
Many thanks for your code simplification suggestion! I have committed and tested this.